chroma_core/lib/service_config.py

```diff
@@ -305,16 +306,16 @@ def _stop_services(self):
                 raise RuntimeError(error)
 
     def _init_pgsql(self, database):
-        rc, out, err = self.shell(["service", "postgresql", "initdb"])
-        if rc != 0:
-            if 'is not empty' not in out:
-                log.error("Failed to initialize postgresql service")
-                log.error("stdout:\n%s" % out)
-                log.error("stderr:\n%s" % err)
-                raise CommandError("service postgresql initdb", rc, out, err)
-            return
-        # Only mess with auth if we've freshly initialized the db
-        self._config_pgsql_auth(database)
+            rc, out, err = self.shell(["service", "postgresql", "initdb"])
```

Comment:
Did these just get spaced out 4 more spaces?

Reply:
Not sure how that happened. Just auto-formatted the file instead.